client-admin/src/actions/index.js

```diff
@@ -163,260 +167,263 @@ export const MATH_FETCH_ERROR = "MATH_FETCH_ERROR";
+const passwordResetInitError = err => {
   return {
     type: PWRESET_INIT_ERROR,
-    data: err,
-  };
-};
+    data: err
+  }
+}
 
-const passwordResetInitPost = (attrs) => {
-  return PolisNet.polisPost("/api/v3/auth/pwresettoken", attrs);
-};
+const passwordResetInitPost = attrs => {
+  return PolisNet.polisPost('/api/v3/auth/pwresettoken', attrs)
+}
 
-export const doPasswordResetInit = (attrs) => {
-  return (dispatch) => {
-    dispatch(passwordResetInitInitiated());
+export const doPasswordResetInit = attrs => {
+  return dispatch => {
+    dispatch(passwordResetInitInitiated())
     return passwordResetInitPost(attrs).then(
       () => {
         setTimeout(() => {
           // Force page to load so we can be sure the password is cleared from memory
           // delay a bit so the cookie has time to set
-          window.location = "/pwresetinit/done";
-        }, 3000);
+          window.location = '/pwresetinit/done'
+        }, 3000)
 
-        return dispatch(passwordResetInitSuccess());
+        return dispatch(passwordResetInitSuccess())
       },
-      (err) => dispatch(passwordResetInitError(err))
-    );
-  };
-};
+      err => dispatch(passwordResetInitError(err))
+    )
+  }
+}
 
 /* passwordReset */
 
 const passwordResetInitiated = () => {
   return {
-    type: PWRESET_INITIATED,
-  };
-};
+    type: PWRESET_INITIATED
+  }
+}
 
 const passwordResetSuccess = () => {
   return {
-    type: PWRESET_SUCCESS,
-  };
-};
+    type: PWRESET_SUCCESS
+  }
+}
 
-const passwordResetError = (err) => {
+const passwordResetError = err => {
   return {
     type: PWRESET_ERROR,
-    data: err,
-  };
-};
+    data: err
+  }
+}
 
-const passwordResetPost = (attrs) => {
-  return PolisNet.polisPost("/api/v3/auth/password", attrs);
-};
+const passwordResetPost = attrs => {
+  return PolisNet.polisPost('/api/v3/auth/password', attrs)
+}
 
-export const doPasswordReset = (attrs) => {
-  return (dispatch) => {
-    dispatch(passwordResetInitiated());
+export const doPasswordReset = attrs => {
+  return dispatch => {
+    dispatch(passwordResetInitiated())
     return passwordResetPost(attrs).then(
       () => {
         setTimeout(() => {
           // Force page to load so we can be sure the password is cleared from memory
           // delay a bit so the cookie has time to set
-          window.location = "/";
-        }, 3000);
+          window.location = '/'
+        }, 3000)
 
-        return dispatch(passwordResetSuccess());
+        return dispatch(passwordResetSuccess())
       },
-      (err) => dispatch(passwordResetError(err))
-    );
-  };
-};
+      err => dispatch(passwordResetError(err))
+    )
+  }
+}
 
 /* facebook */
 
 const facebookSigninInitiated = () => {
   return {
-    type: FACEBOOK_SIGNIN_INITIATED,
-  };
-};
+    type: FACEBOOK_SIGNIN_INITIATED
+  }
+}
 
 const facebookSigninSuccessful = () => {
   return {
-    type: FACEBOOK_SIGNIN_SUCCESSFUL,
-  };
-};
+    type: FACEBOOK_SIGNIN_SUCCESSFUL
+  }
+}
```

Comment:
This func is unused, and I'm being encouraged by linters to remove. That ok?

Reply:
This might indicate a regression, but either way, a good excuse to write e2e tests for facebook login :)